backend/msgpack/writer_mgr.go

```diff
@@ -123,21 +123,15 @@ func (mgr *writerManager) WriteTo(
 		mgr.RUnlock()
 		return errInstanceWriterManagerClosed
 	}
-	multiErr := xerrors.NewMultiError()
-	for _, instance := range instances {
-		id := instance.ID()
-		writer, exists := mgr.writers[id]
-		if !exists {
-			err := fmt.Errorf("writer for instance %s is not found", id)
-			multiErr = multiErr.Add(err)
-			continue
-		}
-		if err := writer.Write(shard, mu, pl); err != nil {
-			multiErr = multiErr.Add(err)
-		}
+	id := instance.ID()
+	writer, exists := mgr.writers[id]
```

Comment:
Actually I've been using `exists` throughout the repo and would prefer consistency if possible.